### ISSUE TITLE
Manage to train this network finally?

### ISSUE THREAD

**Issue by jaypatravali** [hszhao/PSPNet#60]
Hello all,

I have been going through the issues, most of it seem not less than 3 months old. So has anyone succesfully trained this network achieving near equal scores as reported in the paper.

My application is a little different. I attempted on the Cityscapes dataset ( for my use case, only 5 labels instead of 19 as given in the paper) gives me a mIoU of 44.9. My current setup is a 2x12GB TitanX pascal, batch size two, using the pspnet50_ADE20K_473.prototxt (added softmaxloss and datapaths) and the pspnet50_ADE20K.caffemodel as pretrained weights. My inputs were resized as 473x473. 
I know this is not the right way, but just wanted to give it a try.

I have been trying to wrap my head around running this one. Any help is most appreciated.

Thanks!

**Comment by bhadresh74** [comment]
@jaypatravali 
I tried to train the network of my own dataset by adding softmaxloss and datapath.
I have 360X480 input size and just three classes.
I cant get good training accuracy and the loss is around ~87.
Do you think I am missing something?

Please check the below issue link,
https://github.com/hszhao/PSPNet/issues/63

Appreciate your help.
Bee

**Comment by ThienAnh** [comment]
@jaypatravali  @bhadresh74  Can you public script for train?

**Comment by bhadresh74** [comment]
@ThienAnh I have shared it in one of my comments at (https://github.com/hszhao/PSPNet/issues/63) under name train_val.txt

**Comment by ThienAnh** [comment]
@bhadresh74 Thank you so much. I have new problem: How can i training with any crop_size: ex: Default of crop_size of Aug data is 437, but i want to training with crop_size is 200, because my GPU memory is not enough for training with 437. 
Thank for your help.

**Comment by ThienAnh** [comment]
Hi .
Where can i get pretrain model for python?
